pallets/creditcoin/src/lib.rs

```diff
@@ -1198,7 +1230,7 @@ pub mod pallet {
 
 		#[transactional]
 		#[pallet::weight(<T as Config>::WeightInfo::register_transfer_ocw())]
-		pub fn register_funding_transfer(
+		pub fn register_funding_transfer_legacy(
```

Comment:
Because of the rename test coverage is missing. Maybe we could parametrize the tests if you don't want to duplicate them. 

On a related note what is the deprecation schedule for this extrinsic ?